src/components/Home/Home.js

```diff
@@ -19,20 +22,34 @@ function Home() {
     errorFetchingMovies,
   } = useFetchMovies();
 
-  const fetchCfg = useCallback(async () => {
-    const response = await getConfiguration();
-    setUrlPrefix(response.images.base_url + response.images.backdrop_sizes[1]);
-    setIsLoading(false);
-  }, []);
+  const getGenres = ids => {
+    return ids.map(id => {
+      const foundGenre = genres.find(genre => genre.id === id);
+      return foundGenre ? foundGenre.name : null;
+    });
+  };
+
+  const handleMoviePress = currentItem => {
+    const movieGenres = getGenres(currentItem.genre_ids);
+    navigation.navigate('Details', {
```

Comment:
Put this on a constants file, look at https://github.com/moove-it/react-native-template/blob/master/src/constants/navigation.js